README.rst

```diff
@@ -0,0 +1,76 @@
+*pliers*: a python package for automated feature extraction
+===========================================================
+
+|PyPI version fury.io| |pytest| |Coverage Status|
+|DOI:10.1145/3097983.3098075|
+
+Pliers is a Python package for automated extraction of features from
+multimodal stimuli. It provides a unified, standardized interface to
+dozens of different feature extraction tools and services–including many
+state-of-the-art deep learning-based models and content analysis APIs.
+It’s designed to let you rapidly and flexibly extract all kinds of
+useful information from videos, images, audio, and text.
+
+You might benefit from pliers if you need to accomplish any of the
+following tasks (and many others!):
+
+-  Identify objects or faces in a series of images
+-  Transcribe the speech in an audio or video file
+-  Apply sentiment analysis to text
+-  Extract musical features from an audio clip
+-  Apply a part-of-speech tagger to a block of text
+
+Each of the above tasks can typically be accomplished in 2 - 3 lines of
+code with pliers. Combining them *all*–and returning a single,
+standardized, integrated DataFrame as the result–might take a bit more
+work. Say maybe 5 or 6 lines.
+
+In a nutshell, pliers provides a high-level, unified interface to a very
+large number of feature extraction tools that span a wide range of
+modalities.
+
+Documentation
+-------------
+
+The official pliers documentation is quite thorough, and contains a
+comprehensive `quickstart`_, `user guide`_ and complete `API
+Reference`_.
+
+Installation
+------------
+
+Simply use pip to install the latest release:
+
+   pip install pliers
+
+Dependencies
+~~~~~~~~~~~~
+
+By default, installing pliers with pip will only install third-party
+libraries that are essential for pliers to function properly. However,
+because pliers provides interfaces to a large number of feature
+extraction tools, there are dozens of optional dependencies that may be
+required depending on what kinds of features you plan to extract. You
+can choose to install dependencies piece meal (pliers will alert you if
+you’re missing a depedency) or you may install all the required
+dependencies:
+
+   pip install -r optional-dependencies.txt
+
+Note, however, that some of these Python dependencies have their own
+(possibly platform-dependent) requirements. For example, python-magic
+requires libmagic and without this, you’ll be relegated to loading all
+your stims explicitly rather than passing in filenames (i.e.,
+``stim = VideoStim('my_video.mp4')`` will work fine, but passi
```

Comment:
it does look like the README is cut off?!?

Reply:
Wild. I've always suspected @adelavega doesn't know how to operate copy and paste ;)

Reply:
heh

Reply:
sounds right, must be a limit on the website, I guess you need to run pandoc locally.

Reply:
I did now. Now something seems to be failing w/ sphinx grr

Reply:
that's odd: something to do with this changing?
success: https://github.com/PsychoinformaticsLab/pliers/runs/2169953211?check_suite_focus=true#step:5:15
failure: https://github.com/PsychoinformaticsLab/pliers/runs/2186864097#step:5:14

I wonder why that changed? would manually adding `/home/runner/.local/bin` to PATH fix it?

Reply:
That did it. Weird.